backend/Infrastructure/DatabaseContext/ApplicationContext.cs

```diff
@@ -35,23 +35,47 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             .WithOne(dept => dept.Organization);
 
         modelBuilder.Entity<Organization>()
-            .HasMany<Client>(organization => organization.Clients)
```

Comment:
Was there are requirements to have Customers/Clients in org from before?
Why was this removed?

Reply:
Not removed, this line is now mapping absence. Customer is mapped on L41. Git probably marked this as removed since line 37 is identical.